endpoints.json

```diff
@@ -41,5 +41,21 @@
         "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
       }
     }
+  },
+  "/api/articles": {
```

Comment:
make sure to add the http method before the endpoint as in the first few examples so users know what requests they are allowed to make to these endpoints